operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -91,8 +91,9 @@ kiali_defaults:
   kubernetes_config:
     burst: 200
     qps: 175
+
+  cache_config:
```

Comment:
* move the cache_config group up above, below the "auth" group to maintain alphabetical order.
* unless there is a reason not to, define every possible setting in here with their default values. It typically makes the ansible code easier to write and less prone to errors (if, for example, we later need to reference these settings in the ansible scripts).
```
cache_config:
  cache_duration: 30000000
  cache_enabled: false
  istio_objects: false
  kubernetes_objects: false
```